solus_sc/details.py

```diff
@@ -442,7 +442,7 @@ def setup_license_view(self):
         self.view_stack.add_titled(
             lic_wrap, "license", _("License"))
 
-        reportURI = "https://dev.getsol.us/maniphest/task/edit/form/1/"
+        reportURI = "https://github.com/getsolus/packages/issues/new/"
```

Comment:
This can be `issues.getsol.us`

Reply:
Man, I knew there was an URL like that, but I couldn't think of the right one when doing this. Fixed!